generator/cmd/findbasepath.ts

```diff
@@ -0,0 +1,27 @@
+import { executeSynchronous } from '../utils';
+import { findAutogenEntries } from '../autogenlist';
+
+// this function analyzes a branch name and extracts the base path, then proceeds to find it in autogenlist
+// if not found, it throws an error
+executeSynchronous(async () => {
+    if (!process.argv[2]) {
+        throw new Error("Branch name missing. This cmd expects a branch name with the following format: 'sdkAutomation/[basepath]'");
+    }
+    
+    const prName = process.argv[2];
+    let basePath = prName.replace("sdkAutomation/", "");
```

Comment:
Have you discussed this naming convention with the SDK team? Just want to make sure it's going to be robust if we're going to rely on it. What happens if there's a PR spanning multiple directories? Will they know not to change this naming convention?